darshan-util/darshan-logutils.c

```diff
@@ -991,6 +992,51 @@ static int darshan_log_get_namerecs(void *name_rec_buf, int buf_len,
     return(buf_processed);
 }
 
+
```

Comment:
What is the expected format of the log version string?

If it's `<decimal>.<decimal>` we could probably simplify this a lot with `sscanf`?

Something like:

```
ret = sscanf(ver_str, "%d.%d", maj_num, min_num);
if(ret < 2) {
   /* error, it's not formatted right */
}
/* otherwise we are good */
```

Reply:
Good call, I don't know how I ended up with anything other than this implementation :)

See [9ae46b2](https://github.com/darshan-hpc/darshan/pull/726/commits/9ae46b2fbf89b3f1aa7b13cb3f2d3e91a285c229) for satisfying diff of changes.